lib/plugins/aws/utils/formatLambdaLogEvent.js

```diff
@@ -1,25 +1,28 @@
 'use strict';
 
 const dayjs = require('dayjs');
-const chalk = require('chalk');
-const os = require('os');
 const { style } = require('@serverless/utils/log');
 
-module.exports = (msgParam, options = {}) => {
-  const { isModern } = options;
+module.exports = (msgParam) => {
   let msg = msgParam;
-  const dateFormat = 'YYYY-MM-DD HH:mm:ss.SSS (Z)';
+  const dateFormat = 'YYYY-MM-DD HH:mm:ss.SSS';
 
-  if (isModern) {
-    if (!msg.startsWith('REPORT')) msg = msg.trimRight();
-  } else if (msg.startsWith('REPORT')) {
-    msg += os.EOL;
+  if (!msg.startsWith('REPORT')) msg = msg.trimRight();
+
+  if (msg.startsWith('START')) {
```

Comment:
```suggestion
  if (msg.startsWith('START ')) {
```

BTW is that a strict case comparison?

In Bref we have logs like `Starting PHP-FPM` (no prefixes at all), so just making sure about these edge case scenarios.

Reply:
Yes, it's case-aware comparison - I would keep it this way because I believe in some cases the separator used is different than space (in a lot of cases it's `\t`)